charts/common/templates/cron.yaml

```
@@ -43,6 +43,8 @@ spec:
         - name: {{ .name | default $app }}
           image: {{ $image }}
           imagePullPolicy: Always
+          command: {{ .Values.common.container.command }}
+          args: {{ .Values.common.container.args }}
```

We need to add it for the deployment as well. 

Additionally, I think it should only be added if it is set in the values file

---

Thanks. So, something like this? -->

```suggestion
          {{- if .Values.common.container.command -}}
          command: {{ .Values.common.container.command }}
           {{- end }}
           {{- if .Values.common.container.args -}}
           args: {{ .Values.common.container.args }}
           {{- end }}
```

---

I think it's
`.Values.common.container.args` -> `.args`  (and similar for command)

(and indentation slightly off ;) )